apps/condo/domains/common/utils/serverSchema/parseEnvConfigs.js

```diff
@@ -0,0 +1,17 @@
+const conf = require('@open-condo/config')
+const parseConfig = require('@open-condo/config/parse')
```

Comment:
What's the source of this solution? If it's related to `.env` checking, why where's no ADR about this?

It's also strange to validate it here. It should be done in single place, why is it in `serverSchema`?